opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -90,31 +92,45 @@ message LabelKey {
 }
 
 // A collection of data points that describes the time-varying values
-// of a metric.
-message TimeSeries {
+// of a gauge metric.
+message GaugeTimeSeries {
   // TODO: Add restrictions for characters that can be used for keys and values.
   // The set of label values that uniquely identify this timeseries. Apply to all
   // points. The order of label values must match that of label keys in the
   // metric descriptor.
   repeated LabelValue label_values = 1;
 
-  // The data points of this timeseries. Point type MUST match the MetricDescriptor.type, so for
-  // a CUMULATIVE type only cumulative_points are present and for a GAUGE type only gauge_points
-  // are present.
-  repeated GaugePoint gauge_points = 2;
-  repeated CumulativePoint cumulative_points = 3;
+  // The data points of this timeseries. Point type MUST match the MetricDescriptor.type.
+  repeated Point points = 2;
+}
+
+// A collection of data points that describes the time-varying values
+// of a cumulative metric.
+message CumulativeTimeSeries {
+  // The time that the cumulative value was reset to zero.
+  google.protobuf.Timestamp start_time = 1; // required
+
+  // TODO: Add restrictions for characters that can be used for keys and values.
```

Comment:
IIRC we can remove this TODO now, since the decision is not to do any validation.

Reply:
Done.